src/condor_ce_router_defaults

```diff
@@ -158,11 +175,27 @@ JOB_ROUTER_DEFAULTS = """JOB_ROUTER_DEFAULTS_GENERATED @=jrd
                                                    orig_OnExitHoldSubCode,
                                                    1),
                                        42);
+  ]
+  @jrd
+"""
+
+ACCOUNTING_GROUP_CONFIG = """
+JOB_ROUTER_TRANSFORM_AccountingGroup @=jrt
+    # Allow admins to set their own accounting group expressions
+    # that also include the ones set in 'AccountingGroupOSG' (SOFTWARE-2067)
+    SET AccountingGroupOSG {accounting_group}
+    SET AccountingGroup    MY.AccountingGroupOSG
+@jrt
 
-    set_AccountingGroupOSG = @accounting_group@;
+JOB_ROUTER_PRE_ROUTE_TRANSFORMS = $(JOB_ROUTER_PRE_ROUTE_TRANSFORMS) AccountingGroup
+
+JOB_ROUTER_DEFAULTS_GENERATED @=jrd
+  $(JOB_ROUTER_DEFAULTS_GENERATED)
+  [
+    set_AccountingGroupOSG = {accounting_group};
```

Comment:
The previous chunk is in new syntax; should this be in new syntax too? (If not, can you add a comment explaining why?)

Reply:
Nope, the old syntax is still the default so we still need to generate `JOB_ROUTER_DEFAULTS` (which is seeded with `JOB_ROUTER_DEFAULTS_GENERATED`).

Reply:
Done